src/libsync/abstractnetworkjob.h

```diff
@@ -88,6 +88,10 @@ class OWNCLOUDSYNC_EXPORT AbstractNetworkJob : public QObject
      */
     QString errorStringParsingBody(QByteArray *body = 0);
 
+    /** static variable the HTTP timeout (in seconds). If set to 0, the default will be used
+     */
+    static int httpTimeout;
```

Comment:
It would be nice to name variables always with the unit, e.g. httpTimeoutSeconds